.travis.yml

```diff
@@ -12,12 +12,17 @@ matrix:
     python: "nightly"
 
 before_install:
-  - pip install -U ".[dev]"
-  - pip install -U pytest-cov sphinx mypy
+  - pip install -U -r dev-requirements.txt
+  - pip install -U flit>=1.0
```

Comment:
This will need to be >= the next release of flit. I don't know what the release number will be yet, but not `1.0`.

Reply:
I assume that's for the `extras-require` section of `pyproject.toml`?

Reply:
Yeah for the `extras-require` support, and also that `flit install --deps dev` is currently broken in 1.0.